### PR TITLE
LPD-24498 Notification template for an order

### PR DESCRIPTION
**Pull request by smottal**
https://liferay.atlassian.net/browse/LPD-24498

### CODE REVIEW

**Comment by smottal** [comment]
ci:test:sf

File: modules/apps/commerce/commerce-service/src/main/resources/com/liferay/commerce/internal/notification/term/evaluator/dependencies/commerce_order_order_items.ftl [hunk 1]
```diff
@@ -1,80 +1,62 @@
...
-		<tr>
-			<th>
-				${tableLabel}
-			</th>
-		</tr>
-
 		<#foreach orderItem in orderItems>
 			<tr>
-				<td class="order-item-img">
-					<img src="${orderItem.imageURL}">
```
Comment (luca-pellizzon) [inline modules/apps/commerce/commerce-service/src/main/resources/com/liferay/commerce/internal/notification/term/evaluator/dependencies/commerce_order_order_items.ftl:72]: why removing the order item image?

File: modules/apps/commerce/commerce-service/src/main/resources/com/liferay/commerce/internal/notification/term/evaluator/dependencies/commerce_order_order_items.ftl [hunk 2]
```diff
@@ -1,80 +1,62 @@
...
-				</td>
 				<td class="order-item-info">
 					<table>
 						<tr>
-							<th>
+							<th class="order-item-title">
 								${orderItem.name}
 							</th>
 						</tr>
```
Comment (luca-pellizzon) [inline modules/apps/commerce/commerce-service/src/main/resources/com/liferay/commerce/internal/notification/term/evaluator/dependencies/commerce_order_order_items.ftl:62]: aren't we missing order items prices? options, skus, ecc?

**Comment by smottal** [comment]
ci:test:sf

**Review by luca-pellizzon — APPROVED** [review]

**Comment by smottal** [comment]
ci:test:sf

**Comment by smottal** [comment]
ci:test:relevant

**Comment by smottal** [comment]
ci:test:sf

**Comment by smottal** [comment]
ci:test:relevant

**Comment by smottal** [comment]
ci:test:relevant

**Comment by luca-pellizzon** [comment]
ci:forward

**Comment by luca-pellizzon** [comment]
@smottal there are conflicts to solve

**Pull request closed without merging**